apps/mobile/app/helpers/validations.ts

```diff
@@ -1,87 +1,83 @@
-import { IRegisterDataAPI} from "../services/interfaces/IAuthentication"
-import { EMAIL_REGEX, PHONE_REGEX, URL_REGEX } from "./regex";
+import { IRegisterDataAPI } from "../services/interfaces/IAuthentication"
+import { EMAIL_REGEX, PHONE_REGEX, URL_REGEX } from "./regex"
 
-type Err = { [x in keyof IRegisterDataAPI]: string | undefined };
+type Err = { [x in keyof IRegisterDataAPI]: string | undefined }
 
-type Ks = { [x: string]: string };
+type Ks = { [x: string]: string }
 
-export const authFormValidate = (
-  keys: (keyof IRegisterDataAPI)[],
-  values: IRegisterDataAPI
-) => {
-  const err = {} as Err;
-  keys.forEach((key) => {
-    switch (key) {
-      case "email":
-        if (!EMAIL_REGEX.test(values["email"])) {
-          err["email"] = "Please provide a properly formatted email address";
-        }
-        break;
-      case "name":
-        if (values["name"].trim().length < 2) {
-          err["name"] = "You must provide a valid Name";
-        }
-        break;
-      case "recaptcha":
-        if (values["recaptcha"].trim().length < 2) {
-          err["recaptcha"] =
-            "Please check the ReCaptcha checkbox before continue";
-        }
-        break;
-      case "team":
-        if (values["team"].trim().length < 2) {
-          err["team"] = "You must provide a valid Team Name";
-        }
-        break;
-      case "code":
-        if (values["code"].trim().length < 6) {
-          err["code"] = "Your invitation code must contain 6 digits";
-        }
-        break;
-    }
-  });
+export const authFormValidate = (keys: (keyof IRegisterDataAPI)[], values: IRegisterDataAPI) => {
+	const err = {} as Err
+	keys.forEach((key) => {
+		switch (key) {
+			case "email":
+				if (!EMAIL_REGEX.test(values.email)) {
```

Comment:
@desperado1802 make sure we have support of "+" sign in email.

i.e. example+1@domain.com, example+testing@domain.com...